ani.ter-markosyan/repo/Lesson_3/exercise_18/exercise_18.py

```diff
@@ -0,0 +1,5 @@
+radius = int(input("Enter a radius of a circle: "))
+PI = 3.14159
+print("Circle's diameter is", 2 * radius)
+print("Circle's circumference is", round(2 * PI * radius, 2))
+print("Circle's area is", round(PI * radius * radius, 2))
```

Comment:
LT: It would be more correct to represent the multiplication of the same number as a power.